_data/simple-icons.json

```
@@ -8944,6 +8944,23 @@
 		"source": "https://lbry.com/press-kit",
 		"guidelines": "https://lbry.com/faq/acceptable-use-policy"
 	},
+	{
+		"title": "LCEDA",
```

See https://lceda.cn/page/company. Its English name is `EasyEDA`.

```suggestion
		"title": "EasyEDA",
```